### PR TITLE
[13.0] [MIG] stock_picking_comment_template

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

**Comment by feketemihai** [comment]
Depends on https://github.com/OCA/account-invoice-reporting/pull/149

**Comment by feketemihai** [comment]
Depends on https://github.com/OCA/reporting-engine/pull/417

File: stock_picking_comment_template/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* `C2i Change 2 improve <https://www.c2i.es>`_:
```
Comment (emagdalenaC2i) [inline stock_picking_comment_template/readme/CONTRIBUTORS.rst:1]:
```suggestion
* `C2i Change 2 improve <http://www.c2i.es>`_:
```
Please, change this

**Review by emagdalenaC2i — COMMENTED** [review]
I will test it properly once base_comment_template is ready to merge

**Pull request closed without merging**